src/app/pages/jobs/jobs-list/jobs-list.component.ts

```diff
@@ -0,0 +1,77 @@
+import { Component } from '@angular/core';
+import { UntilDestroy } from '@ngneat/until-destroy';
+import { TranslateService } from '@ngx-translate/core';
+import { JobsManagerStore } from 'app/components/common/dialog/jobs-manager/jobs-manager.store';
+import { EntityTableComponent } from 'app/pages/common/entity/entity-table/entity-table.component';
+import { JobRow } from 'app/pages/jobs/jobs-list/job-row.interface';
+import { LocaleService } from 'app/services/locale.service';
+import { T } from 'app/translate-marker';
+import { EntityTableConfig } from '../../common/entity/entity-table/entity-table.interface';
+
+@UntilDestroy()
+@Component({
+  template: '<entity-table [title]="title" [conf]="this"></entity-table>',
+  styleUrls: ['./jobs-list.component.scss'],
+})
+export class JobsListComponent implements EntityTableConfig<JobRow> {
+  title = this.translate.instant(T('Job Log'));
+  queryCall: 'core.get_jobs' = 'core.get_jobs';
+  entityList: EntityTableComponent;
+
+  columns = [
+    {
+      name: T('Name'),
+      prop: 'method',
+      always_display: true,
+      job: true,
+      widget: {
+        component: 'JobItemComponent',
+        icon: 'assignment',
+      },
+    },
+    { name: T('Logs/Errors'), prop: 'logs_excerpt' },
+    { name: T('ID'), prop: 'id', sort: 'desc' },
+    { name: T('Started'), prop: 'date_started' },
+    { name: T('Finished'), prop: 'date_finished' },
+    { name: T('Abortable'), prop: 'abortable' },
+    { name: T('State'), prop: 'state' },
+  ];
+  config = {
+    paging: true,
+    sorting: { columns: this.columns },
+    deleteMsg: {
+      title: 'Name',
+      key_props: ['method'],
+    },
+  };
+
+  constructor(
+    private translate: TranslateService,
+    private localeService: LocaleService,
+    private store: JobsManagerStore,
+  ) {}
+
+  preInit(entityList: EntityTableComponent): void {
+    this.entityList = entityList;
+  }
+
+  resourceTransformIncomingRestData(data: JobRow[]): JobRow[] {
+    return data.map((item) => this.transformJob(item));
+  }
+
+  transformJob(job: JobRow): JobRow {
```

Comment:
Same here, first argument is Job.
You'll get type errors, you can:
- cast `{ ... job } as JobRow`
- or have single return statement:
```
return {
   ...job,
   date_started: ...
}
```